index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="../Device/css/style.css">
+        <title>Интернет магазин Device</title>
+    </head>
+    <body>
+        <header class="header">
+            <div>
+                <a class="nav_link" href="#">devide</a>
+                <input type="text" name="search">
+                <button type="button">Войти</button>
+                <button type="button">Сравнить</button>
```

Comment:
Войти and Сравнить are more links than buttons, because in real life they'll lead to separate pages of login and comparison
And please consider <ul> for this group of elements

Reply:
you should try to go with minimum amount of tags, if no need in div, than don't use it